glide.yaml

```diff
@@ -0,0 +1,4 @@
+package: github.com/theplant/hsm2-backend/vendor/github.com/theplant/resources
```

Comment:
Good question! I just generated it with `glide init`...

I'll force-push a change.